app/components/MovieContainer.js

```diff
@@ -1,31 +1,15 @@
 import React from 'react';
-import Axios from 'axios';
+import { Link } from 'react-router-dom';
 import MovieElement from './MovieElement';
 
-class MovieContainer extends React.Component {
-
-  state = {
-    movies: []
-  }
-
-  componentDidMount() {
-    Axios.get('http://varnatrd.tech/api/series').then(({ data = [] }) => {
-      this.setState({ movies: data });
-    });
-  }
-
-  render() {
-    const { movies } = this.state;
-    return (
-      <div>
-        {movies.map(({ covertImage, title, year }) => {
-          return (
-            <MovieElement imageSource={covertImage} title={title} year={year} />
-          );
-        })}
-      </div>
-    );
-  }
-}
+const MovieContainer = ({ movies }) => (
+  <div>
+    {movies.map(({ covertImage, title, year, _id }) => (
```

Comment:
What would happen if `movies` is just an empty array and you are trying to `destruct` an empty array and pass `undefined` to the `MovieElement` Component? 💣 🔥 🚒  maybe?